File: src/main/java/com/m2t/hachikodictionary/dto/quiz/QuizDto.java
package com.m2t.hachikodictionary.dto.quiz;

import com.m2t.hachikodictionary.model.Account;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@AllArgsConstructor
@NoArgsConstructor
public class QuizDto {
    private String id;
    private Account account;
    private int correctAnswers;
    private int incorrectAnswers;
    private int notAnswered = 0;
    private String difficulty;
}

File: src/main/java/com/m2t/hachikodictionary/dto/QuizDto.kt
package com.m2t.hachikodictionary.dto

import com.m2t.hachikodictionary.model.Account

data class QuizDto(
    val id: String?,
    val account: Account,
    val correctAnswers: Int,
    val incorrectAnswers: Int,
    val notAnswered: Int = 0,
    val difficulty: String
) {
}